### PR TITLE
Add comment integration

### LINKED ISSUE #2439: Add comment integration
Add comment to video [link](https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2213%3A140569)

### PR DESCRIPTION
**Pull request by drillprop**
Fix #2439

### CODE REVIEW

**Comment by kdembler** [comment]
When not logged in :P

Screenshot 2022-05-09 at 17 16 05

**Comment by drillprop** [comment]
> When not logged in :P
> 
> Screenshot 2022-05-09 at 17 16 05

fixed

File: packages/atlas/src/components/_comments/Comment/Comment.styles.ts [hunk 1]
```diff
@@ -68,6 +68,7 @@ export const StyledSvgActionTrash = styled(SvgActionTrash)`
 
 export const CommentBody = styled(Text)`
   display: flex;
+  white-space: pre-line;
   align-items: center;
```
Comment (eldiegod) [inline packages/atlas/src/components/_comments/Comment/Comment.styles.ts:72]: nice

**Review by eldiegod — CHANGES_REQUESTED** [review]
Works as expected but there is an issue with focus/blur in CommentInput; I would guess it's either because of the change in the onFocus functions or by the custom placeholder, I remember I thought it was not worth it to have a custom placeholder and think it was because of this.

To reproduce the issue: click on the CommentInput -> don't type -> click multiple times in the container besides the buttons -> it will case the input to close and open flicker and blur/focus flciker

**Comment by drillprop** [comment]
> I remember I thought it was not worth it to have a custom placeholder and think it was because of this.

Unfortunately, the design requires a custom placeholder:
image
It's impossible to change the styling of one word(in this case "bedeho") using only `::placeholder` selector. I had to add something custom. 

> To reproduce the issue: click on the CommentInput -> don't type -> click multiple times in the container besides the buttons -> it will case the input to close and open flicker and blur/focus flciker

I think I fixed this. Let me know what you think

**Review by eldiegod — APPROVED** [review]
yeah now it's good

**Pull request merged**